README.md

```diff
@@ -90,6 +91,24 @@ Executables here will run just before those in `/opt/odoo/common/build.d`.
 Files here will be environment-variable-expanded and concatenated in
 `/opt/odoo/auto/odoo.conf` at build time.
 
+#### `/opt/odoo/custom/ssh.d`
+
+Files here will be used to create the `root` and `odoo` users' SSH directories.
+
+It should follow the same structure as a standard `~/.ssh` directory, including
+`config` and `known_hosts` files.
```

Comment:
Since we concluded we don't need this for `odoo` user, I'm thinking on a different approach, just add this line to the Dockerfile:

```
RUN ln -s /opt/odoo/custom/ssh ~root/.ssh
```

And we're done!

Possibly this comment invalidates some of my comments below, but I'll keep them just in case. 😅